digiroad2-api-viite/src/main/scala/fi/liikennevirasto/digiroad2/ViiteApi.scala

```diff
@@ -308,23 +308,22 @@ class ViiteApi(val roadLinkService: RoadLinkService, val vVHClient: VVHClient,
   }
 
   put("/roadlinks/roadaddress/project/savenewroadlink") {
-    try { //check for validity
```

Comment:
yea i kinda took the lazy way and just checked first in try catch if i can read values from api message and then read them again in non try-catch